ecs-cli/modules/clients/aws/amimetadata/client_test.go

```diff
@@ -54,7 +43,7 @@ func TestMetadataClient_GetRecommendedECSLinuxAMI(t *testing.T) {
 		},
 		{
 			// validate that we use the generic AMI for other instances
-			"t2.micro",
+			[]string{"t2.micro"},
```

Comment:
since we're accounting for multiple instance types in the other AMI selection tests, can we add a few more "generic" types here? T3, M5a, C4, etc. (doesn't have to be exhaustive, but at least several popular ones that adhere to diff patterns)

Reply:
Added in https://github.com/aws/amazon-ecs-cli/pull/1116/commits/2d6be87aae37b20fd540b9def04ee31ce516ccc9